skimage/registration/tests/test_phase_cross_correlation.py

```diff
@@ -171,16 +171,8 @@ def test_4d_input_subpixel():
     assert_allclose(result, -np.array(subpixel_shift), atol=0.05)
 
 
-@pytest.mark.parametrize("return_error", [True, False, "always"])
 @pytest.mark.parametrize("reference_mask", [None, True])
-def test_phase_cross_correlation_deprecation(return_error, reference_mask):
-    # For now, assert that phase_cross_correlation raises a warning that
-    # returning only shifts is deprecated. In skimage 0.22, this test should be
-    # updated for the deprecation of the return_error parameter.
-    should_warn = (
-        return_error is False
-        or (return_error != "always" and reference_mask is True)
-    )
+def test_phase_cross_correlation_deprecation(reference_mask):
```

Comment:
I wasn't sure if I should just delete this whole function or not.

Reply:
I think we can just get rid of this.